app/src/main/res/layout/header_item.xml

```diff
@@ -32,4 +32,16 @@
         android:gravity="center_vertical"
         tools:text="Game of Thrones" />
 
+    <TextView
+        android:id="@+id/header_more"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginEnd="14dp"
```

Comment:
I wonder if you could extract hardcoded value `14dp` to dimens variable or use one similar which already exists. I suppose also that there is already an item with value `16dp`.

It would help others to easier manage code if there would be more similar views like this.